gradle/project.gradle.kts

```diff
@@ -36,9 +31,15 @@ kotlin {
     experimental.coroutines = Coroutines.ENABLE
 }
 
-allprojects {
-    buildDir = File("Z://build")
+/** working directory of the application */
+val workingDir = file("working dir")
+val kotlinVersion: String by extra {
```

Comment:
Why are the variables here instead top lvl? Proper formatting btw.

Reply:
because usually gradle didnt allow code before the plugins block

Reply:
Is that an issue? Id like to have constants to use for entire script, like kotlin/java version.

Reply:
kotlinVersion is actually defined in the plugins block: `kotlin("jvm") version "1.2.21"`

this is only a way to not having to redefine it. I'll move the other constant back up

Reply:
Ok